docs/minio-client-complete-guide.md

```diff
@@ -1524,20 +1524,20 @@ COMMANDS:
 *Example: Enable versioning on bucket `mybucket`*
 
 ```
-mc versioning enable myminio/mybucket
-myminio/mybucket versioning is enabled
+mc version enable myminio/mybucket
+myminio/mybucket version is enabled
```

Comment:
message should still be `myminio/mybucket versioning is enabled`

Reply:
"versioning" :-)